src/Compilers/CSharp/Test/Semantic/Semantics/PropertyFieldKeywordTests.cs

```diff
@@ -1708,21 +1873,29 @@ public double P
             Assert.Equal(fieldKeywordSymbolInfo, fieldKeywordSymbolInfo2);
             Assert.Equal("System.Double C.<P>k__BackingField", comp.GetTypeByMetadataName("C").GetFieldsToEmit().Single().ToTestDisplayString());
             Assert.Same(comp.GetTypeByMetadataName("C").GetFieldsToEmit().Single(), fieldKeywordSymbolInfo.Symbol.GetSymbol());
-            Assert.Equal(0, accessorBindingData.NumberOfPerformedAccessorBinding);
+
+            if (comp.IsNullableAnalysisEnabledAlways)
```

Comment:
>if (comp.IsNullableAnalysisEnabledAlways) [](http://example.com/codeflow?start=12&length=41)

I think that relying on an environment for these tests is not a good idea. Let's disable that nullable debug validation behavior for all SemanticModel tests for this feature that check ```NumberOfPerformedAccessorBinding```.